client/components/site-selector/index.jsx

```diff
@@ -267,7 +267,7 @@ class SiteSelector extends Component {
 	renderSites() {
 		let sites;
 
-		if ( ! this.props.hasLoadedSites ) {
+		if ( this.props.isRequestingSites ) {
```

Comment:
This reintroduces the bug fixed in #21387. When the network request for all sites is active, placeholder will be shown although we have list of all sites in local storage.

Reply:
Yeah, I've written a comment back then on the PR that this would happen:

> the placeholder wouldn't be shown as there would already be at least a single site in sites.items by the time you open Site Selector.
> If we introduce the loading of recent sites first, we'll redo this condition. Let's merge for now.

I'll think about if it's possible to do it somehow differently, but I think if sites are being loaded, it means we should show the site placeholder, even if there are already some sites loaded and shown in the Site Selector.